src/types/entities/guild.rs

```diff
@@ -33,7 +33,7 @@ pub struct Guild {
     #[cfg_attr(feature = "sqlx", sqlx(skip))]
     pub emojis: Vec<Emoji>,
     #[cfg_attr(feature = "sqlx", sqlx(skip))]
-    pub features: Vec<String>,
+    pub features: String, // TODO: Make this a 'simple-array'
```

Comment:
What is the reason for this change? https://discord.com/developers/docs/resources/guild#get-guild documents `features` as being a json-list of Strings, which matches `Vec<String>` pretty well. I'd rather not add a TODO onto the main branch :P

Reply:
Mainly it's set that way because the data is saved to the database as a comma delimited array, which we have no easy way to convert to a Vec<String> atm

Reply:
can't we use split/splice functions to do this?

Reply:
But on the flip side, this makes it harder to work with the feature-attribute. It is also kind of misrepresenting what the API-data actually is. I am not a fan of this as-is, but if you, for example, added a method get_features() which returns a Vec<String> of features, I could see merging this. Another method would be to implement saving it into the Database as a `String` type in symfonia, if that works.